apps/studio/src/lib/types/session.ts

```diff
@@ -1,13 +1,22 @@
+import type { Tagged } from "type-fest"
 import { type User } from "@prisma/client"
 import { type IronSession } from "iron-session"
-import { Tagged } from "type-fest"
 
+import type { VerificationToken } from "~/server/modules/database"
 import { type SgidSessionProfile } from "~/server/modules/auth/sgid/sgid.utils"
 
 type CurrentUserId = Tagged<User["id"], "CurrentUserId">
 
 export interface SessionData {
   userId?: CurrentUserId
+  singpass?: {
+    sessionState?: {
+      userId: CurrentUserId
```

Comment:
maybe we should update this to another tagged type instead?

Reply:
Updated in e412915736917847ce3db7f6fad006d4ed08acc7.